### PR TITLE
chore: Adding new versioning strategies

### PR DESCRIPTION
**Pull request by ddixit14**
This PR adds two new versioning strategies:

-  always-bump-minor : This VersioningStrategy always bumps the minor version
-  always-bump-major: This VersioningStrategy always bumps the major version

### CODE REVIEW

**Comment by ddixit14** [comment]
@chingor13 Need some insights to write new tests/make the tests pass. 

image

**Comment by suztomo** [comment]
For the new strategy, follow the test cases already written for "always-bump-patch". (I hope the tests are there)

**Comment by chingor13** [comment]
> Need some insights to write new tests/make the tests pass.

We use a snapshot of the `CLI --help` output to test the CLI help feature. Try running `npm run test:snap` which should rewrite the test snapshots.

**Review by suztomo — APPROVED** [review]

**Comment by suztomo** [comment]
@ddixit14 Do you plan to add a test case in versioning-strategies/java.ts ?

**Review by chingor13 — CHANGES_REQUESTED** [review]
Yes, please add unit tests for the new versioning strategies even though they seem trivial.

Otherwise, LGTM.

**Review by chingor13 — APPROVED** [review]

File: test/versioning-strategies/always-bump-minor.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it} from 'mocha';
+
+import {expect} from 'chai';
+import {AlwaysBumpMinor} from '../../src/versioning-strategies/always-bump-minor';
+import {Version} from '../../src/version';
+
+describe('MinorVersioningStrategy', () => {
```
Comment (bcoe) [inline test/versioning-strategies/always-bump-minor.ts:21]: Tiny nit, this should be `AlwaysBumpMinorStrategy`, my bad because I copy pasted this.

**Pull request merged**